### ISSUE TITLE
"with data set" message duplicating in test name

### ISSUE THREAD

**Issue by urmaul** [Codeception/Specify#22]
I have a test with dataProvider and 4 "specify" calls inside. That made test log look really weird:

```
Codeception PHP Testing Framework v2.0.16
Powered by PHPUnit 4.7.7 by Sebastian Bergmann and contributors.

Tests\codeception\common.unit Tests (1) ----------------------------------------------------------------------------------------------
Test parse path | #0 | #0 | #0 | #0 | (tests\codeception\...Test::testParsePath with data set #0 with data set #0 with data set #0 with data set #0 | SPECIFY NAME | examples index 0)Fail
Test parse path | #0 | #0 | #0 | #0  (tests\codeception\...Test::testParsePath with data set #0 with data set #0 with data set #0 with data set #0)Ok
--------------------------------------------------------------------------------------------------------------------------------------


Time: 118 ms, Memory: 12.75Mb

There was 1 failure:

---------
1) tests\codeception\...Test::testParsePath with data set #0 with data set #0 with data set #0 with data set #0 | SPECIFY NAME | examples index 0 with data set #0
Failed asserting that ...
```

Look at that "with data set #0 with data set #0 with data set #0 with data set #0" in test name.
If I comment one "specify" block 0 there's only 3 "with data set #0"'s in test name.

specify version 0.4.3

**Comment by izabolotnev** [comment]
I have same problem.

**Comment by DavertMik** [comment]
Fixed in 1.0
